delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/MigrationModule.scala

```diff
@@ -114,6 +117,14 @@ class MigrationModule(appCfg: AppConfig, config: Config)(implicit classLoader: C
     new OwnerPermissionsScopeInitialization(acls, appCfg.permissions.ownerPermissions, serviceAccount)
   }
 
+  make[SchemaReferenceExchange]
```

Comment:
Each of them should go in the dedicated module no ?

Reply:
good point